README.md

```diff
@@ -1,5 +1,5 @@
 > This README is for babel-loader v8 + Babel v7
-> Check the [7.x branch](https://github.com/babel/babel-loader/tree/7.x) for docs with Babel v6
+> For docs with Babel v6 see [7.x branch](https://github.com/babel/babel-loader/tree/7.x) 
```

Comment:
```suggestion
> If you are using Babel v6, see the [7.x branch](https://github.com/babel/babel-loader/tree/7.x) docs
```

Reply:
What about "If you are using legacy Babel v6"? I really want to drop this line as Babel 8 is to be released 😄

Reply:
Sure